src/interpreter/interpreter.cpp

```diff
@@ -60,8 +65,16 @@ struct ExpressionInterpreter : OverriddenVisitor<ExpressionInterpreter, Flow> {
   Flow visitCallIndirect(CallIndirect* curr) { WASM_UNREACHABLE("TODO"); }
   Flow visitLocalGet(LocalGet* curr) { WASM_UNREACHABLE("TODO"); }
   Flow visitLocalSet(LocalSet* curr) { WASM_UNREACHABLE("TODO"); }
-  Flow visitGlobalGet(GlobalGet* curr) { WASM_UNREACHABLE("TODO"); }
-  Flow visitGlobalSet(GlobalSet* curr) { WASM_UNREACHABLE("TODO"); }
+  Flow visitGlobalGet(GlobalGet* curr) {
+    push(frame().instance.globalValues[curr->name]);
+    return {};
+  }
+  Flow visitGlobalSet(GlobalSet* curr) {
+    auto value = pop();
+    std::cerr << value;
+    frame().instance.globalValues[curr->name] = value;
```

Comment:
This can be a one-liner, too 🤩 
```suggestion
    frame().instance.globalValues[curr->name] = pop();
```

Reply:
done